Misc/NEWS.d/next/Core and Builtins/2022-12-12-05-30-12.gh-issue-100188.sGCSMR.rst

```diff
@@ -0,0 +1 @@
+The ``BINARY_SUBSCR_LIST_INT`` instruction is no longer used for negative ``int``s, since that instruction always misses when encountering negative integers.
```

Comment:
```suggestion
The ``BINARY_SUBSCR_LIST_INT`` instruction is no longer used for negative integers because that instruction always misses when encountering negative integers.
```